src/lib/graphql/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { Resolvers } from '@graphql-poc/graphql';
+import { join } from 'path';
+import { importSchema } from 'graphql-import';
+import { makeExecutableSchema } from 'graphql-tools';
+import * as Query from './resolvers/Query';
+import { GraphQLSchema } from 'graphql';
+
+// @TODO: This will need to be fleshed out when we have more requirements.
```

Comment:
Even though we're not using it currently, This should still probably set to what this value is currently. Even an empty object would be better than `any`.